pkg/apis/controlplane/v1beta2/types.go

```diff
@@ -74,6 +73,14 @@ type GroupMember struct {
 	Ports []NamedPort `json:"ports,omitempty" protobuf:"bytes,4,rep,name=ports"`
 }
 
+// +k8s:deepcopy-gen:interfaces=k8s.io/apimachinery/pkg/runtime.Object
+// GroupMemberList is a list of GroupMember objects that are cuurently selected by a Group.
+type GroupMemberList struct {
```

Comment:
The name may confuse people that it's the list response of GroupMember API.
Maybe name it GroupMembers, in light of Endpoints.

Reply:
Done